src/themes/set-theme.ts

```diff
@@ -35,8 +35,9 @@ export default function setTheme({ name, customTheme }: SetThemeOptions): void {
   }
 
   if (name === "custom") {
-    if (customTheme && customTheme.length > 0) {
-      insertCustomTheme(customTheme);
+    if (customTheme && customTheme.trim().length > 0) {
+      const protectedCustomTheme = "body{opacity:1;}" + customTheme;
```

Comment:
We need to make sure opacity is always set. To avoid any blank screen.
The reason to set the opacity is to show the UI after the initialization in javascript has finished.